db/citations.yml

```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
   wwwemail: ''
   wwwpub: ''
   year: '2022'
+llan;sbm;prm24:
+  ackno: ''
+  author:
+    - Ling Lan
+    - Qiang Du
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: ''
+  grant: DE-SC0024141
+  journal: Physical Review Materials
+  month: tbd
+  nb: ''
+  note: submitted
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: ''
+  synopsis: A quantitative measure of symmetry breaking is introduced that allows the quantification
```

Comment:
synopsis is way shorter.  Something like `introduces the continuous symmetry breaking measure based on Jensen-Shannon divergence" or something.

What you have written could go in under a key of `professional_summary:`